packages/relay/src/lib/eth.ts

```diff
@@ -191,14 +199,25 @@ export class EthImpl implements Eth {
       if (blockCount <= 0) {
         return EthImpl.feeHistoryZeroBlockCountResponse;
       }
-
-      const cacheKey = `${constants.CACHE_KEY.FEE_HISTORY}_${blockCount}_${newestBlock}_${rewardPercentiles?.join('')}`;
-      let feeHistory: object | undefined = this.cache.get(cacheKey);
-      if (!feeHistory) {
-        feeHistory = await this.getFeeHistory(blockCount, newestBlockNumber, latestBlockNumber, rewardPercentiles, requestId);
-        if (newestBlock != EthImpl.blockLatest && newestBlock != EthImpl.blockPending) {
-          this.logger.trace(`${requestIdPrefix} caching ${cacheKey}:${JSON.stringify(feeHistory)} for ${constants.CACHE_TTL.ONE_HOUR} ms`);
-          this.cache.set(cacheKey, feeHistory);
+      let feeHistory: object | undefined;
+      if(this.getEthFeeHistoryFixedFee()) {
```

Comment:
The challenge with this is that `this.translateBlockTag(EthImpl.blockLatest, requestId)` makes a call to get the latest block whereas in this case you are trying to avoid that since today no cases in Mainnet will have a latest block before the fee was changed.

Reply:
So i forgot the `this.translateBlockTag` internally calls another method which utilizes the `eth_blockNumber`.
It also doesn't do a call on number. 
So we should be good, here.

Reply:
the additional next block fee in the `feeHistory.baseFeePerGas` should always be there regardless of the newestBlock input is `latest` or a `number`.
We do need to fetch the actual `latest` block number when receiving `latest` due to the response `oldestBlock` that should be `latest`-`blockCount`.